packages/actions/resources/lang/en/import.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+return [
+
+    'label' => 'Import :label',
+
+    'modal' => [
+
+        'heading' => 'Import :label',
+
+        'form' => [
+
+            'file' => [
+                'label' => 'File',
+                'placeholder' => 'Upload a .csv file',
+            ],
+
+            'columns' => [
+                'label' => 'Columns',
+                'placeholder' => 'Select a column',
+            ],
+
+        ],
+
+        'actions' => [
+
+            'download_example' => [
+                'label' => 'Download an example .csv file.',
+            ],
+
+            'import' => [
+                'label' => 'Start import',
```

Comment:
Does this work as well?

```suggestion
                'label' => 'Import',
```